helm/charts/hpe-cosi-driver/Chart.yaml

```diff
@@ -1,12 +1,12 @@
-# © Copyright 2024 Hewlett Packard Enterprise Development LP
+# © Copyright 2025 Hewlett Packard Enterprise Development LP
 apiVersion: v2
 name: hpe-cosi-driver
 description: A Helm chart for installing the HPE COSI Driver for Kubernetes
 type: application
 version: 1.0.0
```

Comment:
Please increase the chart version. It's not possible to overwrite versions once published.